phpunit.xml.dist

```diff
@@ -57,4 +57,9 @@
 	<testsuite name="backend-mixed">
 		<directory>tests/Backend/Mixed</directory>
 	</testsuite>
+
+	<testsuite name="storage-types-feature">
+		<file>tests/Backend/Workspaces/MetadataFromRedshiftWorkspaceTest.php</file>
```

Comment:
This one to the `backend-redshift-part-1`

Reply:
No, can't do that because these tests need the project with the feature enabled.

Reply:
I think the last idea was to move test to `backend-mixed`. 

I will put them to `backend-snowflake-part-X` and `backend-redshift-part-X`. Because you are testing each backend separately. Tests in mixed testsuites are testing both backend at the same time (using dataproviders to set type of backend).